src/main/java/persistence/entity/EntityLoader.java

```diff
@@ -0,0 +1,5 @@
+package persistence.entity;
+
+public interface EntityLoader {
+    <T> T find(Class<T> entityType, Object id);
```

Comment:
```suggestion
    <T> T load(Class<T> entityType, Object id);
```

외부 소스(데이터베이스, 파일 등)에서 데이터를 불러오는 개념이니 `load`는 어떨까요?